src/utils/units.ts

```
@@ -0,0 +1,93 @@
+ */
+export const cmToFeet = (cm: number): number => {
+  return cm / 30.48;
+};
+
+/**
+ * Format centimeters for display
+ * @param cm - Value in centimeters
+ * @param decimals - Number of decimal places (default: 0)
+ * @returns Formatted string with "cm" suffix
+ */
+export const formatCm = (cm: number, decimals: number = 0): string => {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)